avm/res/search/search-service/tests/e2e/pe/main.test.bicep

```diff
@@ -38,7 +41,7 @@ module nestedDependencies 'dependencies.bicep' = {
     virtualNetworkName: 'dep-${namePrefix}-vnet-${serviceShort}'
     applicationSecurityGroupName: 'dep-${namePrefix}-asg-${serviceShort}'
     storageAccountName: 'dep${namePrefix}st${serviceShort}'
-    keyVaultName: 'dep-${namePrefix}-kv-${serviceShort}'
+    keyVaultName: 'dep-${namePrefix}-kv-${serviceShort}-${substring(uniqueString(baseTime), 0, 3)}'
```

Comment:
Why the need for a random name that will be different for each run?

Reply:
it is not absolutely necessary, it made my local testing easier. But if it is not desired, I can remove it.

Reply:
I'd prefer that as the issue with it is that if I run the worklow once with the removal flag disabled - and then run it again, there's no change it would use the same key vault again (or might even fail). This is already the case with all resources using CMK, but I think we left it at that (so far).

Reply:
Updated and re-tested